cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/RecoverFromStorageCommand.java

```diff
@@ -657,4 +655,27 @@ public void close() {
         }
     }
 
+    private void validateArguments() {
+        Preconditions.checkArgument(getArgCount() >= 2, "Incorrect argument count.");
+        final String container = getArg(1);
+
+        if (!NumberUtils.isNumber(container)) {
+            Preconditions.checkArgument(container.toLowerCase().equals("all"), "Container argument should either be ALL/all or a container id.");
+            Preconditions.checkArgument(getArgCount() == 2, "Incorrect argument count.");
+        } else {
+            final int startContainer = Integer.parseInt(container);
+            Preconditions.checkArgument(startContainer < this.containerCount, "The start container id does not exist. There are %s containers present", this.containerCount);
+            Preconditions.checkArgument(startContainer >= 0, "The start container id must be a positive number.");
+
+            if (getArgCount() >= 3) {
+                Preconditions.checkArgument(NumberUtils.isNumber(getArg(2)), "The end container id must be a number.");
+                Preconditions.checkArgument(getArgCount() == 3, "Incorrect argument count.");
```

Comment:
Condition should be **getArgCount() >= 3, "Incorrect argument count."** instead of **getArgCount() == 3, "Incorrect argument count."**. is it correct?

Reply:
Hi Swetha, this condition validates exactly 3 arguments if its less or more it will throw error. It will not be less as it is already validated so it validates for more 3 and throws error if it is mor than 3.